backend/public-graph/graph/resolver.go

```diff
@@ -2622,14 +2665,18 @@ func (r *Resolver) ProcessPayload(ctx context.Context, sessionSecureID string, e
 
 	projectID := sessionObj.ProjectID
 	hasBeacon := sessionObj.BeaconTime != nil
+
+	loadWorkspaceSettingsSpan, _ := util.StartSpanFromContext(ctx, "public-graph.pushPayload.loadWorkspaceSettings")
```

Comment:
if you return the context and propagate it, redis child spans can be associated to this parent